[2025-08-18T22:30:29.182+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: generate_data.generate_fraud_data manual__2025-08-18T22:30:24.948357+00:00 [queued]>
[2025-08-18T22:30:29.198+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: generate_data.generate_fraud_data manual__2025-08-18T22:30:24.948357+00:00 [queued]>
[2025-08-18T22:30:29.199+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2025-08-18T22:30:29.218+0000] {taskinstance.py:2214} INFO - Executing <Task(_PythonDecoratedOperator): generate_fraud_data> on 2025-08-18 22:30:24.948357+00:00
[2025-08-18T22:30:29.226+0000] {standard_task_runner.py:60} INFO - Started process 359 to run task
[2025-08-18T22:30:29.231+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'generate_data', 'generate_fraud_data', 'manual__2025-08-18T22:30:24.948357+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/generate_data.py', '--cfg-path', '/tmp/tmp23vj0ug1']
[2025-08-18T22:30:29.234+0000] {standard_task_runner.py:88} INFO - Job 6: Subtask generate_fraud_data
[2025-08-18T22:30:29.306+0000] {task_command.py:423} INFO - Running <TaskInstance: generate_data.generate_fraud_data manual__2025-08-18T22:30:24.948357+00:00 [running]> on host 1205caf1711b
[2025-08-18T22:30:29.423+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='generate_data' AIRFLOW_CTX_TASK_ID='generate_fraud_data' AIRFLOW_CTX_EXECUTION_DATE='2025-08-18T22:30:24.948357+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-18T22:30:24.948357+00:00'
[2025-08-18T22:30:29.898+0000] {logging_mixin.py:188} INFO - Error inserting labeled transaction data: 1062 (23000): Duplicate entry '1' for key 'labeled_transactions.PRIMARY'
[2025-08-18T22:30:29.901+0000] {python.py:202} INFO - Done. Returned value was: None
[2025-08-18T22:30:29.918+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=generate_data, task_id=generate_fraud_data, execution_date=20250818T223024, start_date=20250818T223029, end_date=20250818T223029
[2025-08-18T22:30:29.968+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-18T22:30:29.992+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
